src/parameterizations/lateral/MOM_internal_tides.F90

```diff
@@ -1272,110 +1272,111 @@ subroutine propagate_corner_spread(En, energized_wedge, NAngle, speed, dt, G, CS
+          aC = a1 + (a2 + (a3 + a4))
       elseif (0.5*TwoPi <= theta .and. theta < 0.75*TwoPi) then
           xCrn = x(I,J); yCrn = y(I,J)
           ! east area
           a1 = (yE - ySE)*(0.5*(xE + xSE))
           a2 = (ySE - yS)*(0.5*(xSE + xS))
           a3 = (yS - yCrn)*(0.5*(xS + xCrn))
           a4 = (yCrn - yE)*(0.5*(xCrn + xE))
-          aE = a1 + a2 + a3 + a4
+          aE = a1 + (a2 + (a3 + a4))
           ! northeast area
           a1 = (yNE - yE)*(0.5*(xNE + xE))
           a2 = (yE - yCrn)*(0.5*(xE + xCrn))
           a3 = (yCrn - yN)*(0.5*(xCrn + xN))
           a4 = (yN - yNE)*(0.5*(xN + xNE))
-          aNE = a1 + a2 + a3 + a4
+          aNE = a1 + (a2 + (a3 + a4))
           ! north area
           a1 = (yN - yCrn)*(0.5*(xN + xCrn))
           a2 = (yCrn - yW)*(0.5*(xCrn + xW))
           a3 = (yW - yNW)*(0.5*(xW + xNW))
           a4 = (yNW - yN)*(0.5*(xNW + xN))
-          aN = a1 + a2 + a3 + a4
+          aN = a1 + (a2 + (a3 + a4))
           ! area within cell
           a1 = (yCrn - yS)*(0.5*(xCrn + xS))
           a2 = (yS - ySW)*(0.5*(xS + xSW))
           a3 = (ySW - yW)*(0.5*(xSW + xW))
           a4 = (yW - yCrn)*(0.5*(xW + xCrn))
-          aC = a1 + a2 + a3 + a4
+          aC = a1 + (a2 + (a3 + a4))
       elseif (0.75*TwoPi <= theta .and. theta <= 1.00*TwoPi) then
           xCrn = x(I-1,J); yCrn = y(I-1,J)
           ! north area
           a1 = (yNE - yE)*(0.5*(xNE + xE))
           a2 = (yE - yCrn)*(0.5*(xE + xCrn))
           a3 = (yCrn - yN)*(0.5*(xCrn + xN))
           a4 = (yN - yNE)*(0.5*(xN + xNE))
-          aN = a1 + a2 + a3 + a4
+          aN = a1 + (a2 + (a3 + a4))
           ! northwest area
           a1 = (yN - yCrn)*(0.5*(xN + xCrn))
           a2 = (yCrn - yW)*(0.5*(xCrn + xW))
           a3 = (yW - yNW)*(0.5*(xW + xNW))
           a4 = (yNW - yN)*(0.5*(xNW + xN))
-          aNW = a1 + a2 + a3 + a4
+          aNW = a1 + (a2 + (a3 + a4))
           ! west area
           a1 = (yCrn - yS)*(0.5*(xCrn + xS))
           a2 = (yS - ySW)*(0.5*(xS + xSW))
           a3 = (ySW - yW)*(0.5*(xSW + xW))
           a4 = (yW - yCrn)*(0.5*(xW + xCrn))
-          aW = a1 + a2 + a3 + a4
+          aW = a1 + (a2 + (a3 + a4))
           ! area within cell
           a1 = (yE - ySE)*(0.5*(xE + xSE))
           a2 = (ySE - yS)*(0.5*(xSE + xS))
           a3 = (yS - yCrn)*(0.5*(xS + xCrn))
           a4 = (yCrn - yE)*(0.5*(xCrn + xE))
-          aC = a1 + a2 + a3 + a4
+          aC = a1 + (a2 + (a3 + a4))
       endif
 
       ! energy weighting ----------------------------------------
-      a_total = aNE + aN + aNW + aW + aSW + aS + aSE + aE + aC
-      E_new(m) = (aNE*En(i+1,j+1) + aN*En(i,j+1) + aNW*En(i-1,j+1) + &
-                  aW*En(i-1,j) + aSW*En(i-1,j-1) + aS*En(i,j-1) + &
-                  aSE*En(i+1,j-1) + aE*En(i+1,j) + aC*En(i,j)) / (dx(i,j)*dy(i,j))
+      a_total = (((aNE + aN) + (aNW + aW)) + ((aSW + aS) + (aSE + aE))) + aC
+      E_new(m) = ((((aNE*En(i+1,j+1) + aN*En(i,j+1)) + &
```

Comment:
This set of parentheses is not rotationally invariant. If the expressions for the aNE, etc., were rotationally invariant, one rotationally invariant set of parentheses for a_total would be `a_total = (((aNE + aSW) + (aNW+aSE)) + ((aN + aS) + (aW +aE))) + aC)`.